server/container.go

```diff
@@ -597,5 +597,27 @@ func (s *Server) PortForward(ctx context.Context, req *pb.PortForwardRequest) (*
 
 // Status returns the status of the runtime
 func (s *Server) Status(ctx context.Context, req *pb.StatusRequest) (*pb.StatusResponse, error) {
-	return nil, nil
+
+	// Deal with Runtime conditions
+	runtimeReadyConditionString := "RuntimeReady"
+	runtimeReady, err := s.runtime.RuntimeReady()
+	if err != nil {
+		return nil, err
+	}
+	networkReadyConditionString := "NetworkReady"
+	networkReady, err := s.runtime.NetworkReady()
+	if err != nil {
+		return nil, err
+	}
+
+	resp := &pb.StatusResponse{
+		Status: &pb.RuntimeStatus{
+			Conditions: []*pb.RuntimeCondition{
+				&pb.RuntimeCondition{&runtimeReadyConditionString, &runtimeReady, nil, nil, nil},
```

Comment:
Also, lint failures in the build here. We use keyed fields in composite structs.